### DISCUSSION TITLE
Footer as Page-footer, not as content footer

### DISCUSSION THREAD

**Discussion by FlorianReimold** [executablebooks/meta#944] [Q&A]
I am using the books theme for our Eclipse eCAL Documentation (great theme btw 😊):
https://eclipse-ecal.github.io/ecal/

The single thing I really don't like however is, that the footer is included in the article content frame and not as a general page-footer. I feel like this distracts from the content of my page, as now the footer feels like being part of the article-content:

![image](https://user-images.githubusercontent.com/11774314/218961653-4ee76e6c-98d7-4c40-9d00-f70137d5ebf5.png)

**My quesiton now is: How do I change that back?** The [original pydata theme](https://pydata-sphinx-theme.readthedocs.io/en/v0.11.0/user_guide/layout.html#overview-of-theme-layout) only populates the side-wide footer by default and I would like to bring that back.

----

I am using:
- Python 3.8 - 3.10
- Sphinx 4.5
- Sphinx book theme 0.3.3

**Comment by choldgraf** [comment]
I think that doing this will require a little upstream change in the pydata theme, and then we can re-expose that configuration in the book theme as well. I'll see if I can get a quick PR in before the next release.

**Reply by FlorianReimold** [reply]
Oh yea, that would be amazing, thank you very much in advance!
